docs/guides/voyage/voyage.md

```diff
@@ -210,6 +210,10 @@ Voyages have specific requirements, milestones, and ***require a commitment of 8
 
      All participants in a team will be from the same tier. This is the tier you selected on the [Voyage Signup form](https://forms.gle/NkSP7DS3oz4sEzdh8).
 
+     * Tier 1: Developers, plus Agile Product Owners and/or Scrum Masters if they are available
+     * Tier 2: All roles
```

Comment:
shouldn't all tier be "PO and/or SM" if they are available and we didn't want them in tier 1?

Reply:
@cherylli @onetoughcookie226 and I discussed this the other day and decided that for Tier 1 we would add PO and or SM if they are available since they can be helpful in guiding new WebDev's to success. We didn't include the other roles since we wanted to Tier 1's focus on refining and building technology skills.

Reply:
I see. I thought tier 1 would get overwelmed by PO and SM, even for tier 2/3, all these agile things feel too much to learn for a 6 week project. I guess it depends on how strict the PO/SM are following agile. I think they (tier 1) would work better with UI/UX since most of them has no design experience.

Reply:
Thanks. I've updated to make the tier composition clearer and added how we'll do it for Data Science teams as well.